beacon-chain/dberr/errors.go

```diff
@@ -0,0 +1,6 @@
+package dberr
```

Comment:
it seems odd to create a new package just for `dberr` under the `beacon-chain` level, can we move it to `beacon-chain/db/error` ?

Reply:
Actually I forgot that the error values I started putting in the db package had already been merged down to develop. In fact `db.ErrNotFound` already exists. I'll use those instead.